metabase.gemspec

```
@@ -24,8 +24,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.5'
 
-  spec.add_runtime_dependency 'faraday', '>= 1.2.0'
-  spec.add_runtime_dependency 'faraday_middleware'
+  spec.add_runtime_dependency 'faraday', '>= 2.12.2'
```

```suggestion
  spec.add_runtime_dependency 'faraday', '~> 2.0'
```

Кажется, что нет смысла фиксировать так строго версию. Для гема нужна просто версия 2.x, поэтому предлагаю так и указать здесь.

---

++